src/main/java/dev/codesupport/web/api/data/repository/ContributorRepository.java

```diff
@@ -0,0 +1,16 @@
+package dev.codesupport.web.api.data.repository;
+
+import dev.codesupport.web.api.data.entity.ContributorEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ContributorRepository extends JpaRepository<ContributorEntity, Long> {
+
+    //S00100 - Underscores are spring syntax, and are required
+    @SuppressWarnings("squid:S00100")
+    List<ContributorEntity> findAllByContributorList_Id(Long id);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'findAllByContributorList_Id' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5197432)